src/python/gudhi/datasets/remote.py

```diff
@@ -0,0 +1,206 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Hind Montassif
+#
+# Copyright (C) 2021 Inria
+#
+# Modification(s):
+#   - YYYY/MM Author: Description of the modification
+
+from os.path import join, split, exists, expanduser
+from os import makedirs, remove, environ
+
+from urllib.request import urlretrieve
+import hashlib
+import shutil
+
+import numpy as np
+
+def get_data_home(data_home = None):
```

Comment:
I wonder how useful it is for this function to be public. When do we expect users will use it directly? And when do we expect a non-default argument to be used?

Reply:
Yeah I was wondering about that too... But since you mentioned it, I made the function private.